.github/workflows/release-please.yml

```diff
@@ -1,79 +1,55 @@
+name: Release
 on:
   push:
     branches:
       - master
-name: release-please
+permissions:
+  contents: write
+  pull-requests: write
 jobs:
   release-please:
+    name: Release Please
     runs-on: ubuntu-latest
-    outputs:
-      release_created: ${{ steps.release.outputs.release_created }}
-      sha: ${{ steps.release.outputs.sha }}
-      upload_url: ${{ steps.release.outputs.upload_url }}
-    # https://github.com/google-github-actions/release-please-action in case more config is needed
     steps:
       - uses: google-github-actions/release-please-action@v3
         id: release
         with:
+          release-type: node
           command: manifest
-          # In case we need to add a PAT and have the pr's trigger actions
-          #token: ${{ secrets.PAT }}
-          # See how to configure file https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md
-          # See output info at https://github.com/google-github-actions/release-please-action#configuration for variables
-          # Trigger more steps using "if: ${{ steps.release.outputs.release_created }}"
-
+    outputs:
+      tag: ${{ steps.release.outputs.tag_name }}
+      release_created: ${{ steps.release.outputs.release_created }}
+      sha: ${{ steps.release.outputs.sha }}
+      upload_url: ${{ steps.release.outputs.upload_url }}
+    # Need to allow GitHub Actions to create and approve pull requests.
```

Comment:
What is this comment about?

Reply:
Just a simple tip, I can delete it if you don't think it's necessary.